app/helpers/assessment_query_helper.rb

```diff
@@ -61,43 +61,76 @@ def sort(assessments, order, direction)
   end
 
   # Paginates assessments data.
-  def paginate(assessments, entries, page)
+  def self.paginate(assessments, entries, page)
     return assessments if entries.blank? || entries <= 0 || page.blank? || page.negative?
 
     assessments.paginate(per_page: entries, page: page + 1)
   end
 
   # Formats assessments to be displayed on the frontend.
-  def format_for_frontend(assessments)
-    table_data = []
-    symptoms = Assessment.get_unique_symptoms_for_assessments(assessments.pluck(:id))
+  def self.format_for_frontend(assessments)
```

Comment:
I was looking deeper into the micro benchmark that was made and there seems to be something strange going on with the `format_for_frontend` method. It seems to consistently give blank results (as if no assessments were passed to the function. 

## Right before the assessments loop
![Screen Shot 2021-04-20 at 6 38 18 AM](https://user-images.githubusercontent.com/24628687/115397140-ff9a2c80-a1a2-11eb-9708-94245c95707a.png)

## What is being returned by the function
![Screen Shot 2021-04-20 at 6 38 49 AM](https://user-images.githubusercontent.com/24628687/115397196-12acfc80-a1a3-11eb-99e1-9d9c7cfe5e49.png)

Reply:
putting a byebug at the top of the method shows that assessments seem to still be intact as they are passed to `format_for_frontend`

![Screen Shot 2021-04-20 at 6 40 31 AM](https://user-images.githubusercontent.com/24628687/115397417-4ee05d00-a1a3-11eb-9c58-cb8a0f88d343.png)

Reply:
The root of the issue appears to be in the `assessment_ids = assessments.map(&:id)` line

Reply:
@timwongj changing the `paginate` page argument to 0  fixes this. It looks like that method was written to always add `1` to the page number that passed - so probably expecting page 1 to be index 0. Page 2 in this case would definitely have 0 entries since all 100 are shown on page 1

Reply:
made the change here: https://github.com/SaraAlert/SaraAlert/pull/806/commits/c593b41a33decfcecc95ce7cb6d875fadcc59129#diff-eb2f94faa1282056a764ceeae2b4bb03b361611302fd04d7b0ba3b3949573ec4R66

thanks for debugging and catching this!